Public/Src/Sandbox/MacOs/projects.dsc

```diff
@@ -21,13 +21,23 @@ namespace Sandbox {
         derivedDataOutDir: StaticDirectory
     }
 
-    const sandboxSealDir = Transformer.sealSourceDirectory(
-        d`${Context.getMount("Sandbox").path}`,
-        Transformer.SealSourceDirectoryOption.allDirectories);
-
-    const thirdPartySealDir = Transformer.sealSourceDirectory(
-        d`../../../../third_party`,
-        Transformer.SealSourceDirectoryOption.allDirectories);
+    function sourceFileDependencies()
```

Comment:
If this function is called multiple times, it will seal those directories multiple times (once per invocation).

That's why I had those as variables before.

```suggestion
    const sourceFileDependencies = (() =>
```

Reply:
Good catch!